docs/sql/statements/end.rst

```diff
@@ -33,5 +33,5 @@ Description
 
 The statement commits the current transaction.
 
-As CrateDB does not support transactions, this command has no effect and will
-be ignored.
+As CrateDB does not support transactions, the only effect of this command is
+to close all existing cursors ``WITHOUT HOLD`` in the current session.
```

Comment:
in the current `transaction` isn't it more correct?

Reply:
Technically it would be, but because we only support one "transaction" per session and in the same sentence we are saying we do not support transactions I worry it may get confusing. Otherwise, maybe we could say something like, END completes a transaction but transactions in CrateDB only concern the lifetime of cursors?

Reply:
Yeah, it can lead to unnecessary confusion, I agree.